Source/NSDrawer.m

```diff
@@ -92,6 +99,35 @@ + (void) initialize
     }
 }
 
+- (void) _configureContainer: (NSRect)contentRect
+{
+  NSRect rect = contentRect;
+  NSSize containerContentSize;
+
+  rect.origin.x += 1.0;
+  rect.origin.y -= 5.0;
+  if (_container == nil)
+    {
+      _container = [[NSBox alloc] initWithFrame: rect];
```

Comment:
I think this is not related to your change, but `_container` never gets released. This should happen in the dealloc method.

Reply:
I'll fix that.  I'm working on a small change Riccardo and I discussed yesterday.

Reply:
It is now released when it is added to the subview.   This is slightly better since the parent view retains it.